cmd/maya-apiserver/cstor-operator/spc/storagepool_create.go

```diff
@@ -163,27 +163,25 @@ func (pc *PoolCreateConfig) withDisks(casPool *apis.CasPool, spc *apis.StoragePo
 	if err != nil {
 		return nil, errors.Wrapf(err, "aborting storagepool create operation as no node qualified")
 	}
-	if len(nodeBDs.BlockDevices.Items) == 0 {
-		return nil, errors.New("aborting storagepool create operation as no block device was found")
+
+	claimedNodeBDs, err := pc.ClaimBlockDevice(nodeBDs, spc)
```

Comment:
Let us be consistent in naming -- either use `BD` or `BlockDevice` at places unless some exceptions!

Reply:
and some places `disk` also used